app/Website/src/Http/Controllers/NewsletterSubscribeController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Website\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Http;
+use Knuckles\Scribe\Attributes\Endpoint;
+use App\Website\Http\Requests\SubscribeRequest;
+use Illuminate\Support\Facades\RateLimiter;
+
+class NewsletterSubscribeController extends Controller
+{
+    #[Endpoint('Website/Subscribe', <<<'DESC'
+  This endpoint is for subscribing to Leanhub.mk.
+  Additionally it uses Cloudflare Turnstile ReCaptcha for validation.
+ DESC)]
+    public function subscribe(SubscribeRequest $request) {
+
+        $first_name = $request->input('first_name');
+        $email = $request->input('email');
+
+        $executed = RateLimiter::attempt(
```

Comment:
You can add the `throttle` middleware in the route instead, as you are not doing anything custom here. `->middleware('throttle:5,1')` should do the work.

Reply:
Added